### PR TITLE
Nirbhay/Added Support for network manager global network resource

### PR DESCRIPTION
**Pull request by Nirbhay1997**
Signed-off-by: Nirbhay Kumar <nkumar@progress.com>

### Description

Please describe what this change achieves. Ensure you have read the [Contributing to InSpec AWS](https://github.com/inspec/inspec-aws/CONTRIBUTING.md) document before submitting.

### Issues Resolved

List any existing issues this PR resolves, or any Discourse or StackOverflow discussion that's relevant

### Check List
Please fill box or appropriate ([x]) or mark N/A.
- [x] New functionality includes integration tests/controls
- [x] New Terraform resources
- [x] Documentation provided or updated for resources 
- [x] All Integration Tests pass
- [ ] All Unit Tests pass
- [x] `rake lint` passes
- [x] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco>

### CODE REVIEW

File: docs/resources/aws_network_manager_global_networks.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+---
+title: About the aws_network_manager_global_networks Resource
+platform: aws
+---
+
+# aws_network_manager_global_networks
+
+Use the `aws_network_manager_global_networks` InSpec audit resource to test properties of a single specific AWS Network Manager Global Network.
```
Comment (soumyo13) [inline docs/resources/aws_network_manager_global_networks.md:8]: Change this description.

**Review by soumyo13 — CHANGES_REQUESTED** [review]
Add Terraform.

**Comment by Nirbhay1997** [comment]
No Terraform is available for the resource : https://github.com/hashicorp/terraform-provider-aws/issues/11132

**Comment by Nirbhay1997** [comment]
(byebug) @aws.network_manager_client.describe_global_networks(pagination_options)

*** Seahorse::Client::NetworkingError Exception: Failed to open TCP connection to networkmanager.xxxx-2.amazonaws.com:xxx (getaddrinfo: Name or service not known)
**_### Probably a bug in ruby sdk_**

**Review by soumyo13 — APPROVED** [review]
LGTM

**Review by IanMadd — APPROVED** [review]

**Pull request merged**